### PR TITLE
created catalog table

### LINKED ISSUE #8: Create Table for Catalog Table
## Instructions
- Using the DB model provided, write a `CREATE TABLE` statement in SQL for the Catalog table.
- Drop the table if it already exists.

## Notes
- For now, assume that all attributes should be `NOT NULL` unless otherwise specified.
- Arrows on the DB model indicate foreign key constraints
- For any `ENUM` fields, make sure to define them in a line above the `CREATE TABLE` statement.

## Acceptance Criteria
- SQL creation code should exist in a file located in `server/schema/catalog.sql`

## Resources
- [DB Model](https://www.figma.com/file/NDPdo1MR13qgsB4LubtzyZ/Database-Model?type=whiteboard&node-id=0-1&t=m1J0vcLI2mzIb5SN-0)
- [Create Table Tutorial](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-create-table/)
- [Creating Enums Tutorial](https://www.postgresql.org/docs/current/datatype-enum.html)

### PR DESCRIPTION
**Pull request by kristenyee**
closes #8

### CODE REVIEW

**Review by michellelin1 — DISMISSED** [review]
LGTM! Good work :))

File: server/schema/catalog.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+CREATE TYPE event AS ENUM ('guest speaker', 'study-trip', 'workshop');
+CREATE TYPE subject AS ENUM ('life skills', 'science', 'technology', 'engineering', 'math', 'college readiness');
+CREATE TYPE year AS ENUM ('junior', 'senior', 'both');
+
+DROP TABLE catalog IF EXISTS;
```
Comment (michellelin1) [inline server/schema/catalog.sql]: y'all are really close! but one minor change. i'm pretty sure that it's `DROP TABLE IF EXISTS [table name]`. After you change that your all set

Comment (kristenyee) [reply server/schema/catalog.sql]: fixed! mb

**Review by ThatMegamind — APPROVED** [review]
Looks good!

**Pull request merged**